app/src/main/java/com/peacecorps/pcsa/circle_of_trust/CircleOfTrustFragment.java

```diff
@@ -0,0 +1,120 @@
+package com.peacecorps.pcsa.circle_of_trust;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.telephony.SmsManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import com.peacecorps.pcsa.R;
+
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class CircleOfTrustFragment extends Fragment {
+    ImageButton requestButton;
+    ImageButton editButton;
+
+    private String optionSelected;
+    public CircleOfTrustFragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_circle_of_trust, container, false);
+        requestButton = (ImageButton) rootView.findViewById(R.id.requestButton);
+        editButton = (ImageButton) rootView.findViewById(R.id.editButton);
+        editButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                startActivity(new Intent(getActivity(),Trustees.class));
+            }
+        });
+        requestButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showOptions();
+            }
+        });
+        return rootView;
+    }
+
+    public void showOptions(){
+        AlertDialog.Builder builderSingle = new AlertDialog.Builder(
+                getActivity());
+        builderSingle.setTitle("Select a request");
+        final ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.select_dialog_singlechoice);
+        arrayAdapter.add("Come get me");
```

Comment:
It is not a good practice to use hard coded strings in the middle of your program. Specially in Android. I have mentioned you the fact before too. Add the strings in string.xml and use a reference to that.